### PR TITLE
Repository sync with upstream from InterUSS

### PR DESCRIPTION
**Pull request by fmo00**

### CODE REVIEW

File: build/dev/read_version.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Retrieve token from dummy OAuth server
+ACCESS_TOKEN=$(curl --silent \
```
Comment (Ramboio) [inline build/dev/read_version.sh:6]: This token should be obtained from Kong Auth

**Comment by Ramboio** [comment]
The OpenAPI def of the versioning API should be included as well: https://github.com/interuss/automated_testing_interfaces/tree/main/versioning

My suggestion is to copy the definition statically, to the folder /interfaces/versioning/

As for the openapi-to-go-server, I think the versioning OpenAPI should NOT be included in the makefile "dss_apis", because this specific endpoint is not meant to change frequently, so it would just add overhead to the code generation. When one wants to update the versioning schema, the code generation should be done by hand

**Comment by fmo00** [comment]
> The OpenAPI def of the versioning API should be included as well: https://github.com/interuss/automated_testing_interfaces/tree/main/versioning
> 
> My suggestion is to copy the definition statically, to the folder /interfaces/versioning/
> 
> As for the openapi-to-go-server, I think the versioning OpenAPI should NOT be included in the makefile "dss_apis", because this specific endpoint is not meant to change frequently, so it would just add overhead to the code generation. When one wants to update the versioning schema, the code generation should be done by hand

Done ✅

File: build/dev/read_version.sh (hunk 1 above)
Comment (fmo00) [reply build/dev/read_version.sh:6]: Done ✅

**Review by Ramboio — APPROVED** [review]

**Pull request merged**